### PR TITLE
Session 3 Assignment PR

### PR DESCRIPTION
**Pull request by markhilsisonsayson**

### CODE REVIEW

File: Session3/EFCoreAssignment.Services/ProductService.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
+﻿namespace EFCoreAssignment.Data.Services
+{
+    public class ProductService : IProductService
+    {
+
+        public ProductService() { }
+
+        public async Task<List<ProductDto>> GetProducts()
+        {
```
Comment (ranton) [inline Session3/EFCoreAssignment.Services/ProductService.cs:17]: The Services is not Implemented

**Review by ranton — COMMENTED** [review]
You added Session4 assignment here.. that is okay but please double check next time

File: Session3/EFCoreAssignment.Services/ProductService.cs (hunk 1 above)
Comment (markhilsisonsayson) [reply Session3/EFCoreAssignment.Services/ProductService.cs:17]: Done Sir Thanks!

**Comment by ranton** [comment]
All good! thank you

**Pull request merged**